tools/epanet-output/src/outputapi.h

```diff
@@ -12,50 +12,50 @@
 #define MAXFNAME     259   // Max characters in file name
 #define MAXID         31   // Max characters in ID name
 
-// This is an opaque struct. Do not access variables.
-typedef struct ENResultsAPI ENResultsAPI;
+// This is an opaque pointer to struct. Do not access variables.
+typedef void* ENR_Handle;
 
-typedef enum {
+typedef enum elmType {
```

Comment:
is this really needed - adding an explicit enum name, when it's presumably never used?

Reply:
Agreed. I will make this change.